app/Resources/views/base.html.twig

```diff
@@ -151,5 +151,7 @@
 
             <script src="{{ asset('js/app.js') }}"></script>
         {% endblock %}
+
+        <!-- Page rendered on {{ "now"|date('j/M/Y H:i:s') }} -->
```

Comment:
I don't care much because this is for developers instead of end-users. But considering that this improvement comes "for free" I've implemented your suggestion. Thanks!